web/README.md

```diff
@@ -1,10 +1,28 @@
-# VAST Docs
+# VAST Website
 
-This directory the [official VAST documentation][site], along with the
-[Docusaurus](https://docusaurus.io/) scaffold for local building, viewing, and
-deploying.
+This directory contains code and content of the official VAST website at
+https://vast.io. We use [Docusaurus](https://docusaurus.io/) as driving
+framework and an [enhanced flavor of Markdown](TODO) for primary content.
```

Comment:
https://docs.github.com/en/get-started/writing-on-github is a good resource on GHFM.

Reply:
Yeah, but I'd rather get folks to read the Docusaurus docs, as there's more than GFM to consider.